towhee/operator/base.py

```diff
@@ -119,6 +131,27 @@ def framework(self):
     def framework(self, framework: str):
         self._framework = framework
 
+    def predict(self, data):
+        model = self.model if not hasattr(self, '_model_client') else self._model_client
+        return model(data)
+
+    def initialize(self, key, extra):
+        super().initialize(key)
```

Comment:
What does `initialize` do actually?

Reply:
Something needs to do when the object is created.